build/gulp/tasks/stats.ts

```diff
@@ -121,3 +123,27 @@ task(
     'build:stats:bundle',
   ),
 )
+
+task('stats:save', async () => {
+  const commandLineArgs = _.pick(argv, ['sha', 'branch', 'tag', 'pr', 'build'])
+  const bundleStats = require(paths.docsSrc('bundleStats.json'))[UNRELEASED_VERSION_STRING]
+
+  const options = {
+    method: 'POST',
+    uri: process.env.STATS_URI,
+    body: {
+      sha: process.env.CIRCLE_SHA1,
+      branch: process.env.CIRCLE_BRANCH,
+      tag: process.env.CIRCLE_TAG, // optional
+      pr: process.env.CIRCLE_PULL_REQUEST, // optional
+      build: process.env.CIRCLE_BUILD_NUM,
+      ...commandLineArgs, // allow command line overwrites
+      bundleSize: bundleStats,
+      ts: new Date(),
+    },
+    json: true,
+  }
+
+  const ret = await rp(options)
```

Comment:
lets omit shortcuts here, as it makes the code a bit unreadable. We could end up with something like (with import aliasing and variable being renamed)

```js
const response = await requestHttp(options)
```